Assets/MixedRealityToolkit/Extensions/VectorExtensions.cs

```diff
@@ -162,6 +162,56 @@ public static bool IsValidVector(this Vector3 vector)
                    !float.IsInfinity(vector.x) && !float.IsInfinity(vector.y) && !float.IsInfinity(vector.z);
         }
 
+        /// <summary>
+        /// Calculates which side a point in world space is on of a plane
+        /// </summary>
+        /// <param name="initialDirection">The normal direction orthagonal to the plane. </param>
+        /// <param name="initialPosition">The point representing the plane's origin</param>
+        /// <returns>true when the compared point is on the other side of the plane</returns>
+        public static bool IsOtherSideOfPoint(this Vector3 vector, Vector3 initialDirection, Vector3 initialPosition)
```

Comment:
the name is a bit confusing. I suggest:

IsOtherSideOfPlane(this Vector3 vector, Vector3 normal, Vector3 planeOrigin)

Reply:
Or even return int instead
negative on opposite side
zero on plane
positive on plane side

Reply:
Should I just remove this now that @Troy-Ferrell has enlightened me on the `Plane` class? :)